app/src/main/java/com/ihunda/android/binauralbeat/BBeatApp.java

```diff
@@ -0,0 +1,22 @@
+package com.ihunda.android.binauralbeat;
```

Comment:
Why do we need this BBeatApp class?

Reply:
BBeatApp Class is for initialising database at the start of app and make its object available to all the activities.

It can be done in BBeat also, but then we need to open / close the database object everytime.

Using Application level class is more efficient